test/unit/action/channel.spec.js

```diff
@@ -265,16 +265,6 @@ describe('Action Channels Unit Tests', () => {
       expect(nav.goChannels, 'was called once');
       expect(channel.closeChannel, 'was called with', {
         channelPoint: 'some-channel-point',
-        force: false,
-      });
-    });
-
-    it('should force close pending-closing channel', async () => {
```

Comment:
Is this for the case where the user has a `pending-close` channel and closes it again?